ipmof/io.py

```diff
@@ -0,0 +1,141 @@
+# IPMOF file input/output library
+# Date: August 2016
+# Author: Kutay B. Sezginel
+import os
+
+
+class babel:
+    """
+    Openbabel read / write functions.
+    """
+    ob = __import__('openbabel')
+
```

Comment:
If module is imported as a class variable then python raises error when parsing, but if it's loaded in __init__ or any function of the class then parsing is completed even though library is not installed.
Moved all imports under read function for both Ase and Babel.